aries-site/src/pages/templates/forms.mdx

```diff
@@ -23,105 +41,72 @@ import { Example } from '../../layouts';
+## When not to use
 
-Within a form, you should:
+Complicated processes where the user might need guidance at various stages, or when the form would be intimidatingly long. In these cases use a [Wizard](https://design-system.hpe.design/templates/wizard?q=error#guidance).
 
-- Use title case and capitalize short prepositions when they are the last word in a sentence (e.g. Sign Up, Sign Out)
-- For headings and captions, use sentence case
 
-### Button placement and labeling
+## Do's
 
-Submit, reset, or cancel buttons should always be placed at the bottom of a Form and be left-aligned.
+**Required fields**
+- Ask only for the required information. Keep the fields to a minimum to allow the user to finish their task as quickly as possible.
+- List only the most common field requirements below the relevant field. 
+- Use asterisks for required fields when the form has more than two required fields and/or has optional fields. 
+- Asterisks for required fields should be the same color as label field. 
+- If a field is optional, write "Optional" next to the form field label.
 
-Button labels should be specific to the action of the button. For example, a [sign-up form](#sign-up) may have a submit button with label "Sign Up". Alternately, a [password change form](#change-password) may have a button with label "Update Password".
+**Format and fields**
+- User-entered input must be encoded to prevent [SQL injection](https://www.w3schools.com/sql/sql_injection.asp) and [XSS attacks](https://en.wikipedia.org/wiki/Cross-site_scripting).
+- Wrap all input elements within a FormField `<FormField label="Username"><TextInput /></FormField>`  
+- Reveal fields progressively so the user is not overwhelmed.
+- Write in sentence case (capitalize the first letter and all following letters should be lowercase).  
+- Fill in default values for as many fields as possible. 
+- Use clear, concise title labels for each field. 
+- Write helpful placeholder text inside the fields. 
+- If the user would likely have doubts, include helpful text below the field, such as [password requirements](#signup) or [additional resources](#customize) on a field's context.
+- Place the submit, reset, and cancel buttons at the bottom left of the form. [See responsive button behavior](https://design-system.hpe.design/components/button?q=button#responsiveness). 
+- Use specific labels on buttons that tell the user what action to expect. 
 
-Button label specificity provides users confidence in the action they are performing.
+**Actions after the user submits information**
+- Provide feedback when the form is submitted successfully (e.g. notification, success page, green check mark, etc.). 
+- [In the case of an error](#requiredoptional), red highlight the field where the error occurred. 
+- List below the button all relevant errors and the corrective action the user must take. 
```

Comment:
In our examples, I believe we are doing errors inline with the formfield when possible and above the submit button otherwise.

Reply:
- List all relevant errors and the corrective action the user must take. Place in-line when possible. When not possible, list above the submit button.

Reply:
Yes, perfect. Small thing -- we've been typing as "inline".